### PR TITLE
initial test fix

### PR DESCRIPTION
**Pull request by C0DK**
### description

added a pull request template.
The markdown can be tested with this tool: https://dillinger.io/
I'm not completly sure that this will use the template on github. it's hard to test without having it in master ^^ however i followed [this guide](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) which should verify that this does in fact work. 

### Changelog
- Added pull request template

### Issue solved
- [HCLANG-1](http://hcl.codes/jira/browse/HCLANG-1)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+_**Please edit this to reflect the pull request. A pullr equest with this line in it will not(!!!) be accepted.**_
+### Description
+A short explaination of the pullrequest if needed. could be "we added support for Python" or a long ass text. idealy a few lines.
```
Comment (nillas12) [inline .github/pull_request_template.md]: Pull request is two words. Words after . should be caps. Ideally"

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ - this has aswell 
+   - which includes this
+   - and also this
+
+### Issues solved 
+add these with the respective key as HCLANG-1 (as an example)
+ - HCLANG-[number]
+
+### Issues opened as a result of this 
+Add something here if new issues arrise because of this. For instance if new features-needs or bugs arrise from this PR. if this is not the cause, then remove this section
```
Comment (nillas12) [inline .github/pull_request_template.md]: arise, arise, features, needs or bugs. . word capsififed

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## FOR REVIEWER [Do not remove]
+A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
+ - Did the PR actually fix the issue
+ - Comment anything that doesn't make sense or anything you want clarified. Many comments are always excuseable
```
Comment (nillas12) [inline .github/pull_request_template.md]: "Comment on anything"    Maybe "always ok" ikke "always excuseable"

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ - Make sure this is ready for exams if this is going into master
+ - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.
+
+### Regarding code [if applicable]
+- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
+- Is public methods documented?
```
Comment (nillas12) [inline .github/pull_request_template.md]: Are public methods

**Comment by C0DK** [comment]
could @technight give this a look. to make sure his Kotlin experience agrees with my points regarding code review??

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -13,18 +13,18 @@ add these with the respective key as HCLANG-1 (as an example)
  - HCLANG-[number]
 
 ### Issues opened as a result of this 
-Add something here if new issues arrise because of this. For instance if new features-needs or bugs arrise from this PR. if this is not the cause, then remove this section
+Add something here if new issues arise because of this. For instance if new features-requests or bugs arise from this PR. If this is not the cause, then remove this section.
```
Comment (nillas12) [inline .github/pull_request_template.md]: Case, ikke cause

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+## FOR REVIEWER [Do not remove]
+A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
+ - Did the PR actually fix the issue
+ - Comment anything that doesn't make sense or anything you want clarified. Many comments are always excuseable
+ - Make sure this is ready for exams if this is going into master
+ - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.
+
+### Regarding code [if applicable]
+- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
```
Comment (technight) [inline .github/pull_request_template.md]: I think CI's should ensure that code conventions are being followed

File: .github/pull_request_template.md (hunk 5 above)
Comment (C0DK) [reply .github/pull_request_template.md]: true

File: .github/pull_request_template.md (hunk 6 above)
Comment (C0DK) [reply .github/pull_request_template.md]: it's always good to have a second pair of eyes on the code i'd say. like ideally yes but it's nice to have an extra pair of eyes making sure that it not complete garbish

**Pull request merged**